addon/components/power-select.hbs

```diff
@@ -120,49 +122,49 @@
         {{/let}}
       {{/if}}
       {{#if this.mustShowSearchMessage}}
-        {{#let 
-          (if 
-            @searchMessageComponent 
+        {{#let
+          (if
+            @searchMessageComponent
             (component (ensure-safe-component @searchMessageComponent))
             (component 'power-select/search-message')
-          ) 
+          )
         as |SearchMessage|}}
-          <SearchMessage 
+          <SearchMessage
             @searchMessage={{this.searchMessage}}
             @select={{publicAPI}}
-            id={{listboxId}} 
-            aria-label={{@ariaLabel}}
-            aria-labelledby={{@ariaLabelledBy}} 
-          /> 
+            id={{listboxId}}
+            @ariaLabel={{@ariaLabel}}
+            @ariaLabelledBy={{@ariaLabelledBy}}
+          />
         {{/let}}
       {{else if this.mustShowNoMessages}}
-        {{#let 
-          (if 
+        {{#let
+          (if
             @noMatchesMessageComponent
             (component (ensure-safe-component @noMatchesMessageComponent))
             (component 'power-select/no-matches-message')
-          ) 
+          )
          as |NoMatchesMessage|}}
-          <NoMatchesMessage 
-            @noMatchesMessage={{this.noMatchesMessage}} 
-            @select={{publicAPI}} 
-            id={{listboxId}} 
-            aria-label={{@ariaLabel}}
-            aria-labelledby={{@ariaLabelledBy}} 
+          <NoMatchesMessage
+            @noMatchesMessage={{this.noMatchesMessage}}
+            @select={{publicAPI}}
+            id={{listboxId}}
+            @ariaLabel={{@ariaLabel}}
```

Comment:
passing through accessible labels, listbox should have the labels associated with them too according to the audit